meta-resin-common/recipes-connectivity/openvpn/openvpn/prepare-openvpn.service

```diff
@@ -9,4 +9,4 @@ RemainAfterExit=yes
 ExecStart=@BINDIR@/prepare-openvpn
 
 [Install]
-WantedBy=resin.target
+WantedBy=config-json.service
```

Comment:
I'm not sure how is this supposed to work. Is adds a wanted dependency on preparevpn for config-json. But the prepare-openvpn would have been already activated. It's confusing to me.